bns-test/test/name_revoke.test.ts

```diff
@@ -0,0 +1,334 @@
+          cases[0].namespace, 
+          "bob", 
+          cases[0].salt, 
+          cases[0].zonefile, { sender: cases[0].nameOwner });
+        expect(receipt.result).include('true');
+        expect(receipt.success).eq(true);
+      // });
+  
+      // describe("Charlie revoking 'bob.blockstack' on Bob's behalf", () => {
+
+      //   it("should fail", async () => {
+          
+          receipt = await bns.nameRevoke(
+            cases[0].namespace, 
+            "bob", { sender: charlie });
+          expect(receipt.error).include('2006');
+          expect(receipt.success).eq(false);
+        // });
+
+        // it("should resolve as expected", async () => {
+          receipt = await bns.getNameZonefile(
+            cases[0].namespace, 
+            "bob", { sender: cases[0].nameOwner });
+          expect(receipt.result).include('0x30303030');
+          expect(receipt.success).eq(true);
+      //   });
+      // });
+
+      // describe("Bob revoking 'bob.blockstack'", () => {
+
+      //   it("should succeed", async () => {
+          
+          receipt = await bns.nameRevoke(
+            cases[0].namespace, 
+            "bob", { sender: cases[0].nameOwner });
+          expect(receipt.result).include('true');
+          expect(receipt.success).eq(true);
+        // });
+
+        // it("should not be able to resolve", async () => {
+          receipt = await bns.getNameZonefile(
+            cases[0].namespace, 
+            "bob", { sender: cases[0].nameOwner });
+          expect(receipt.error).include('2014');
+          expect(receipt.success).eq(false);
+        // });
+
+        // it("Bob trying to update 'bob.blockstack' should fail", async () => {
+          receipt = await bns.nameUpdate(
+            cases[0].namespace, 
+            "bob", 
+            "4444", { sender: bob });
+          expect(receipt.error).include('2014');
+          expect(receipt.success).eq(false);
+        // });
+
+        // it("Bob trying to transfer 'bob.blockstack' should fail", async () => {
+          receipt = await bns.nameTransfer(
+            cases[0].namespace, 
+            "bob", 
+            charlie,
+            "4444", { sender: bob });
+          expect(receipt.error).include('2014');
+          expect(receipt.success).eq(false);
+        // });
+
+        // it("Bob trying to register 'bob.blockstack' should fail", async () => {
+          receipt = await bns.namePreorder(
+            cases[0].namespace,
+            "bob",
+            "salt#2", 
+            2560000, { sender: bob });
+          expect(receipt.success).eq(true);
+          expect(receipt.result).include('u24');
+  
+          receipt = await bns.nameRegister(
+            cases[0].namespace, 
+            "bob", 
+            "salt#2",
+            cases[0].zonefile, { sender: bob });
+          expect(receipt.error).include('2004');
+          expect(receipt.success).eq(false);
+        // });
+
+        return 
```

Comment:
Are the rest of these tests supposed to be ignored due to this early `return`?

Reply:
The test suite was cleaned and fixed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/24ce63a46e9c82b3018a498d9778873f8af0db2f